doc/source/whatsnew/v0.22.0.txt

```diff
@@ -119,6 +119,56 @@ Current Behavior
 
     s.rank(na_option='top')
 
+.. _whatsnew_0220.enhancements.assign_dependent:
```

Comment:
rebase on master. can you move to 0.23 (docs were renamed), prob easiest to just check this file from master and past in new one

Reply:
(also 0220 -> 0230)

Reply:
I think I'll manage to rebase until Christmas.
the whatsnew page I constructed contains the traceback (since a change between python versions is shown) Is this desired? If not, is there a way to just print the code and output as in the document and not run the code actually?